src/main/java/basic/Exercise04.java

```diff
@@ -0,0 +1,27 @@
+package basic;
+
+public class Exercise04 {
+
+    public String findFibonacciSequence(int num) {
+        if (num < 2) {
+            return "";
+        } else {
+            String result = "";
+
+            int fibo1 = 1;
+            int fibo2 = 1;
+            int fibo3 = fibo1 + fibo2;
+
+            result += "" + fibo1;
```

Comment:
Sao không thế này cho gọn

```suggestion
            String result = "" + fibo1;
```

Reply:
done